src/components/pages/order/OrderPage.js

```diff
@@ -4,18 +4,35 @@ import styled from "styled-components";
 import { theme } from "../../../theme";
 import logoOrange from "../../../assets/logo-orange.png";
 import Menu from "./main/Menu";
+import { useState } from "react";
+import GlobalContext from "../../../context/GlobalContext";
 
 export default function OrderPage() {
   const { name } = useParams();
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+  const [panelTabIndex, setPanelTabIndex] = useState(2);
+  const [isPanelReduce, setIsPanelReduce] = useState(false);
+
+  const contextValue = {
```

Comment:
```suggestion
  const globalContextValue = {
```
OPTIONNEL : préférer peut-être globalContextValue au cas où tu aurais un autre context qui vient

Reply:
mieux je suis d'accord